### PR TITLE
SCTE parser for MPEG-TS

### PR DESCRIPTION
**Pull request by elv-serban**
Add support for parsing a SCTE-35 data stream in MPEG-TS, if present.
SCTE-35 signals are aligned with the video stream.

Bonus:
- connect input stats for video and audio first decoded frame / PTS

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -6,23 +6,28 @@ SUBDIRS=utils libavpipe exc elvxc
...
 
 dynamic: copy_libs_all all
 
-clean: lclean
+clean: lclean ## Clean C and Go build artifacts
 	@for dir in $(SUBDIRS); do \
 	echo "Making $@ in $$dir..."; \
 	(cd $$dir; make $@) || exit 1; \
 	done
+	@echo "Go clean"
```
Comment (elv-reza) [inline Makefile:29]: Not a good idea to clean go cache and everything in clean tag (it increase the build time),

It is better to add a new tag, like:

cleanall: clean goclean
  
goclean:
             @echo "Go clean"
             @go clean -modcache -testcache -cache -i -r

**Review by elv-reza — COMMENTED** [review]
@serban, please remove ts directory, we don't use that code and was a kind of prototype.

File: Makefile (hunk 1 above)
Comment (elv-serban) [reply Makefile]: Ok - I reverted the changes to Makefile since they are not necessary or related to this PR.  I can make the Go clean in a separate PR.

**Review by elv-reza — APPROVED** [review]
Looks good!

**Pull request merged**